lib/message_generator.ex

```diff
@@ -2,7 +2,13 @@ defmodule ExcellentMigrations.MessageGenerator do
   @moduledoc false
 
   def build_message(%{type: type, path: path, line: line}) do
-    "#{build_message(type)} in #{path}:#{line}"
+    """
+    #{build_message(type)} in #{path}:#{line}
+
+        For more info:
+          * https://github.com/Artur-Sulej/excellent_migrations#checks
+          * https://github.com/Artur-Sulej/excellent_migrations#assuring-safety
```

Comment:
The link fragments here are inherently brittle, but if the README sections do change without the appropriate update here, the user will still end up on the README document.